### DISCUSSION TITLE
Qdomyos as running speed sensor

### DISCUSSION THREAD

**Discussion by dancov73** [cagnulein/qdomyos-zwift#1586] [Ideas]
it would be nice if qdomyos could be recognized by garmin watches as running speed sensor while connected to treadmill.

**Comment by cagnulein** [comment]
thanks i will do in the next weeks!

Il giorno sab 29 lug 2023 alle 07:13 dancov73 ***@***.***> ha
scritto:

> it would be nice if qdomyos could be recognized by garmin watches as
> running speed sensor while connected to treadmill.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/cagnulein/qdomyos-zwift/discussions/1586>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAALYWHNTCC6HJKQNXDVM3LXSSLW7ANCNFSM6AAAAAA24IS5KA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Roberto Viola
Software engineer and open source enthusiast
http://robertoviola.cloud

**Comment by cagnulein** [comment]
@dancov73 done! https://github.com/cagnulein/qdomyos-zwift/pull/1589 
I will test it and then I will publish on the beta of the play store.
I will let you know

**Reply by dancov73** [reply]
thank you

**Reply by cagnulein** [reply]
@dancov73 test went fine on zwift! 
send me an email to roberto.viola83 at gmail.com indicating this ticket and I will send you the newest beta! Let me know
